horace_core/sqw/file_io/@binfile_v2_common/define_upgrade_map_.m

```diff
@@ -0,0 +1,61 @@
+function obj=define_upgrade_map_(obj,file_exist,old_ldr)
```

Comment:
be good to have a description of what this method does, also for arguments

Reply:
added